x-pack/plugins/alerting/server/rules_client/common/audit_events.ts

```diff
@@ -161,7 +161,12 @@ export function ruleAuditEvent({
   outcome,
   error,
 }: RuleAuditEventParams): AuditEvent {
-  const doc = savedObject ? `rule [id=${savedObject.id}]` : 'a rule';
+  const doc = savedObject
+    ? [`rule [id=${savedObject.id}]`, savedObject.name && `[name:${savedObject.name}]`]
```

Comment:
I believe it is nice to be consistent with our formatting. In the ID we have `=` but in the name `:`. I would say to go with `=` for the name. For example `[name=${savedObject.name}]`